src/scss/includes/components/panel.scss

```
@@ -41,8 +41,8 @@
     overflow-y: auto;
```

`scrollbar-width` is defined on `.poi_panel` (used by the old favorites panel)
I would recommend to set it here too, to replace default scrollbars on Firefox.

```suggestion
     overflow-y: auto;
     scrollbar-width: thin;
```